release-notes/1.19.0.md

```diff
@@ -0,0 +1,91 @@
+## Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.19.0.
+
+Please see [component versions](https://anbox-cloud.io/docs/ref/component-versions) for a list of updated components.
+
+## New features & improvements
+
+### Core stack improvements
+
+* The Anbox Management Service (AMS) API documentation created using OpenAPI specification (Swagger 2.0) is available on the AMS host at `/1.0/swagger.json`.<!--AC-1474-->
```

Comment:
... but also in the official documentation (we need to add a link once the release is out)

Reply:
Yes, I will add it once we have the link generated.